translations.properties

```diff
@@ -47,6 +47,7 @@ webapp/ProfileHeader.properties=webapp/portlet/src/main/resources/locale/portlet
 webapp/ProfileWorkExperience.properties=webapp/portlet/src/main/resources/locale/portlet/social/ProfileWorkExperience_en.properties
 webapp/SpaceAdministrationPortlet.properties=webapp/portlet/src/main/resources/locale/portlet/social/SpacesAdministrationPortlet_en.properties
 webapp/SpacesListApplication.properties=webapp/portlet/src/main/resources/locale/portlet/social/SpacesListApplication_en.properties
+webapp/UserPopup.properties=webapp/portlet/src/main/resources/locale/portlet/social/UserPopup_en.properties
```

Comment:
IMO we should do the same for the spacePopover

Reply:
next step. My PR is still draft.
The spacePopup will be also in this PR